src/config/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""
+__init__.py -   # FIXME teplates
+
+
+
+Copyright (c) 2019 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+
+import pluggy
+
+# load our hook specification
+from ._constants import _PLUGIN_NAME
+from . import _spec
+
+# create the plugin manager
+plugin_manager = pluggy.PluginManager(_PLUGIN_NAME)
```

Comment:
hmm.... this one may require discussion as it is not constant.

yes, the name is constant insofar as it always points to the same object, but the object to which it points to is not constant (due to registration activities)

Reply:
I feel like as long as the value of this particular variable (i.e. the *pointer* to what object this variable references) doesn't change, that it's still technically a constant.
It doesn't necessarily mean that absolutely nothing about it can ever change.

If we were to destroy and re-create it, changing the object ID, *then* it would be a problem.

Reply:
renamed.